app/models/resource.rb

```diff
@@ -5,6 +5,6 @@ class Resource < ActiveRecord::Base
   private
 
   def used_less_than_quantity
-    errors.add(:used, 'can not be higher than total quantity') unless used <= quantity
+    errors.add(:used, 'can not be higher than total quantity') if used.present? && quantity.present? && !(used <= quantity)
```

Comment:
Why do we want to throw this error if  used equals quantity?
&& used > quantity